sdk/storage/storage-blob/test/retrypolicy.spec.ts

```diff
@@ -157,4 +157,29 @@ describe("RetryPolicy", () => {
 
     assert.deepStrictEqual(URLBuilder.parse(finalRequestURL).getHost(), secondaryHost);
   });
+
+  it("Retry Policy should work when on PARSE_ERROR with unclosed root tag", async () => {
+    let injectCounter = 0;
+    const injector = new InjectorPolicyFactory(() => {
+      if (injectCounter === 0) {
+        injectCounter++;
```

Comment:
Where is the check that is supposed to make sure that this code is hit, i.e., the counter is incremented?

Reply:
Good point. I copied an existing test which doesn't verify this. Will add.